pkgs/by-name/pu/pushlog/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3Packages,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "pushlog";
+  version = "0.3.1";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "serpent213";
+    repo = "pushlog";
+    rev = "v${version}";
+    sha256 = "sha256:1cfag0vfi07vnww5sls630xqc41rnyaqzrabia0fwhiill3k6dfm";
```

SHA256 hashes like these are deprecated. Use the hash attribute to get a "new-style" SRI hash (`nix hash convert` also does the job)

```suggestion
    hash = "";
```